### ISSUE TITLE
[icetray] Error in documentation of I3Frame (Trac #2192)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2184]
http://software.icecube.wisc.edu/documentation/projects/icetray/classes/i3frame.html#in-python

The methode "I3Frame.GetStop()" is not available (anymore?), instead there is the attribute "I3Frame.Stop".

Migrated from https://code.icecube.wisc.edu/ticket/2192
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:18", 
    "description": "http://software.icecube.wisc.edu/documentation/projects/icetray/classes/i3frame.html#in-python\n\nThe methode \"I3Frame.GetStop()\" is not available (anymore?), instead there is the attribute \"I3Frame.Stop\".", 
    "reporter": "thomas.kittler", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067318169976", 
    "component": "combo core", 
    "summary": "[icetray] Error in documentation of I3Frame", 
    "priority": "trivial", 
    "keywords": "", 
    "time": "2018-10-01T12:59:55", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "complaint"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20181001T13:48:27: 
* olivas changed status from "new" to "accepted"
* olivas changed milestone from "" to "Autumnal Equinox 2018"
* olivas changed owner from "" to "olivas"
* olivas changed component from "other" to "icecube offline"

**Comment by icecube-trac** [comment]
Trac update at 20181001T21:17:57: olivas changed summary from "Error in documentation of I3Frame" to "[icetray] Error in documentation of I3Frame"

**Comment by icecube-trac** [comment]
Trac update at 20181002T19:13:54: olivas commented:

> In our defense those docs have only been wrong for anywhere from 7-9 years, depending on how you count.  Stop was added as a property 9 years ago.  The method GetStop was changed to get_stop 7 years ago.

**Comment by icecube-trac** [comment]
Trac update at 20181002T19:36:27: olivas commented:

> Turns out the interface changed as well.  9 years ago frame.GetStop() would return a Stop because it was bound to I3Frame::GetStop() whereas now it's bound to the overload of that method that takes a string (frame key) as an argument.

**Comment by icecube-trac** [comment]
Trac update at 20181002T19:54:45: 
* olivas commented:

> In changeset 3105:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="3105"
> updated documentation.  closes #2184.
> ```


* olivas changed status from "accepted" to "closed"
* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:05:30: nwhitehorn commented:

> In changeset 168864:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168864"
> updated documentation.  closes #2184.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:18: 
* olivas changed milestone from "Autumnal Equinox 2018" to ""
* olivas commented:

> Milestone Autumnal Equinox 2018 deleted
